packages/open-api-gateway/test/project/codegen/components/generated-java-client-source-code.test.ts

```diff
@@ -28,4 +28,8 @@ describe("Generated Java Client Code Unit Tests", () => {
   it("Multi", () => {
     expect(synthGeneratedCode("multi.yaml")).toMatchSnapshot();
   });
+
+  it("Single Tags", () => {
+    expect(synthGeneratedCode("single-tags.yaml")).toMatchSnapshot();
+  });
 });
```

Comment:
What does the error look like if use inputs "multiple tags"? Are we able to catch this early in the build and output detailed error indicating multiple is not supported at this time?

Can we add a unit test for multiple tags as well, which is expected to throw an error with actionable messaging?

Reply:
Good call! The error currently would be after all the code is generated (`npx projen`), and would surface as a build error for the generated client (eg defining two methods with the same name due to operations being repeated across different APIs). To test that case properly we'd need to bring back the verdaccio inegration tests I think.

I think we could catch this at `npx projen` time by adding some validation on the openapi spec (for smithy this would be after the openapi spec is generated) - I'll have a look and see what I can do :)

Reply:
Added some validation :)